MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/EnggDiffractionPresenter.h

```diff
@@ -182,7 +182,7 @@ protected slots:
                               Mantid::API::MatrixWorkspace_sptr &vanCurvesWS);
 
   // plots workspace according to the user selection
-  void plotFocusedWorkspace(std::string outWSName, std::string bank);
+  void plotFocusedWorkspace(std::string outWSName, int bank);
```

Comment:
If you used `size_t` here instead of `int` you wouldn't need the `static_cast` in EnggDiffractionPresenter.cpp:1129